src/utils.ts

```diff
@@ -105,6 +109,10 @@ const addAdminJsAuthRoutes = (admin: AdminJS, router: Router, auth: KoaAuthOptio
   })
 
   router.use(async (ctx: ParameterizedContext, next) => {
+    if (ctx.session == null) {
```

Comment:
Just a question since I haven't used koa in my projects. Will this change allow to setup the admin panel without authentication or will it throw an error when it's configured without auth?

Reply:
Yes, this works fine. `addAdminJsAuthRoutes` is only called from `buildAuthenticatedRouter`, which sets up the session just before the call. 

This can be verified by running the example app as well.